bin/-tarball-info.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+/* eslint-disable no-console, node/no-extraneous-require, node/no-unpublished-require */
+const fs = require('fs');
+const path = require('path');
+
+const projectRoot = path.resolve(__dirname, '../');
```

Comment:
Can you write a short comment explaining what the overall purpose of this file is

Reply:
Added docs